data/packages/com.gamebase.scene.yml

```diff
@@ -0,0 +1,17 @@
+name: com.gamebase.scene
+displayName: Gamebase-SceneManagement
+description: 'Gamebase-SceneManagement is manage scene and make scene lifecycle. Gamebase-SceneManagement is an extension for Gamebase-Core and won't work without it.'
```

Comment:
```
description: 'Gamebase-... won't work without it.'
=>
description: "Gamebase-... won't work without it."
```

Escape issue. Use double quote to fix. Is this text generated by the package add form? Then it's a bug.